.github/workflows/project-automation.yml

```diff
@@ -0,0 +1,25 @@
+name: "Project Board Automation"
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+jobs:
+  join_issue_pr_to_project:
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Automate adding issues to Backlog"
+      uses: docker://takanabe/github-actions-automate-projects:v0.0.1
+      if: github.event_name == 'issues'
+      env:
+        GITHUB_TOKEN: ${{ secrets.ORG_GITHUB_TOKEN }}
+        GITHUB_PROJECT_URL: https://github.com/orgs/creativecommons/projects/10
+        GITHUB_PROJECT_COLUMN_NAME: "Pending Review"
+    - name: "Automate adding PRs to Active Sprint"
+      uses: docker://takanabe/github-actions-automate-projects:v0.0.1
+      if: github.event_name == 'pull_request'
+      continue-on-error: true
```

Comment:
This seems strange. If it's not set, what can happen?

Reply:
PRs from outside contributors will error out because they don't have permissions to add things to our organization's projects.

See discussion on https://github.com/creativecommons/cccatalog-api/issues/456

Reply:
oh ok